news/_posts/2025-03-30-release-20250330.md

```diff
@@ -0,0 +1,95 @@
+---
+
+#### Hotfix 20250330: Smarter Harvesters, Bug Fixes, and Smoother Battles
+
+<figure>
+   <video autoplay loop muted>
+    <source src="{{ '/images/news/20250330-harvesters.mp4' | relative_url }}" type="video/mp4" alt="harvesters harvesting ore">
+    <img src="{{ '/images/news/20250330-harvesters.png' | relative_url }}" alt="harvesters harvesting ore" />
+  </video>
+  <figcaption>Harvester operations have been returned to normal</figcaption>
+</figure>
+
+No battle plan survives first contact, and the same goes for software updates. We’re rolling out a hotfix to address a few issues identified in the latest release. Here’s what’s been fixed:
+
+- **Harvesters now mine smarter;** They’ll stick closer to their refinery instead of wandering off
+- **Rally points display correctly;** Unit move commands are now consistently shown
+- **Jungle trees won’t crash the game as they burn;** Some Tiberian Dawn trees were feeling spicy
+- **OpenRA again runs on Raspberry Pi;** Shader compilers be crazy
+- **Lobby alignments fixed;** In multiplayer skirmish lobbies, team numbers are now properly centered
+- **Adjusted Dune 2000 weapon splash damage;** To better match the vanilla balance
+- **Fixed green gliches on Dune 2000 corpses;** Soldiers were spilling green blood
+
+#### A Look Back at OpenRA Release 20250303
+
+The 20250303 release was one of OpenRA’s biggest updates yet, bringing over a year of development and community playtesting into a polished final version. If you missed it, here’s a quick refresher on the major additions:
+
+<figure>
+   <video autoplay loop muted>
+    <source src="{{ '/images/news/20241116-chrono.mp4' | relative_url }}" type="video/mp4" alt="chrono vortex">
+    <source src="{{ '/images/news/20241116-chrono.webm' | relative_url }}" type="video/webm" alt="chrono vortex">
+    <img src="{{ '/images/news/20241116-vortex.webp' | relative_url }}" alt="chrono vortex" />
+  </video>
+  <figcaption>OpenRA now supports post-processing effects out of the box!</figcaption>
+</figure>
+
+We significantly improved support for high-definition art assets, especially for the Tiberian Dawn HD project. This brings:
+* Faster load times
+* Better performance
+* Fewer graphical bugs
+
+Tiberian Dawn HD is packaged as a separate release and can be [downloaded here](https://github.com/OpenRA/TiberianDawnHD/releases/tag/release-20250330). This preview is multiplayer-compatible with the main 20250330 release.
+
+To use it, the C&C Remastered Collection must be installed via Steam or the EA App. If you’re on macOS or Linux, follow the project README for [detailed installation instructions](https://github.com/OpenRA/TiberianDawnHD#asset-installation).
+
+#### Dune 2000 Visual Overhaul
+
+The original Dune 2000 shipped with an option for 16-bit sprites, but OpenRA had previously only supported lower-quality 8-bit versions. Thanks to HD graphics improvements, the game has now been fully upgraded to 16-bit assets, bringing:
```

Comment:
Actually, 99% of the R16 sprites are still only 8 bit - just with per-sprite palettes. Reword to just say "higher quality"?